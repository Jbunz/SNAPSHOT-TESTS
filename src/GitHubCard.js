import { Card } from 'react-bootstrap';

const GitHubCard = ({ username, imageUrl, blurb }) => {
  return (
    <Card>
      <Card.Img variant="top" src={imageUrl} style={{ width: '275px', height: '300px' }} />
      <Card.Body>
        <Card.Title>{username}</Card.Title>
        <Card.Text>{blurb}</Card.Text>
      </Card.Body>
    </Card>
  );
};

export default GitHubCard;
