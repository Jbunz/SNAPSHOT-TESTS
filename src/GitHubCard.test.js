// GitHubCard.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import GitHubCard from './GitHubCard';

test('renders GitHubCard snapshot', () => {
  const tree = renderer
    .create(
      <GitHubCard
        username="Jbunz"
        imageUrl="C:\Users\jacob\Downloads\charlesdeluvio-K4mSJ7kc0As-unsplash.jpg"
        blurb="Photos make me nervous!"
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
