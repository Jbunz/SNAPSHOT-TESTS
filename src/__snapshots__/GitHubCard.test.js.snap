// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GitHubCard snapshot 1`] = `
<div
  className="card"
>
  <img
    className="card-img-top"
    src="C:\\\\Users\\\\jacob\\\\Downloads\\\\charlesdeluvio-K4mSJ7kc0As-unsplash.jpg"
    style={
      Object {
        "height": "300px",
        "width": "275px",
      }
    }
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Jbunz
    </div>
    <p
      className="card-text"
    >
      Photos make me nervous!
    </p>
  </div>
</div>
`;
